make/config.mk

```diff
@@ -132,10 +132,19 @@ endif
 ARCH := $(shell uname -a)
 ifneq (,$(filter $(ARCH), armv6l armv7l powerpc64le ppc64le aarch64))
 	USE_SSE=0
+	USE_F16C=0
 else
 	USE_SSE=1
 endif
 
+#----------------------------
+# F16C instruction support for faster arithmetic of fp16 on CPU
+#----------------------------
+# For distributed training with fp16, this helps even if training on GPUs
+# If left empty, attempts to identify support and turn it on.
+# During cross compilation, please check support for F16C on target device.
+USE_F16C =
```

Comment:
Value missing?

Reply:
If left empty, attempts to identify support and turn it on